Adds 2 Columns - Comment Status and Created Date

kyledodge opened the pull request:
Re-implementing PR changes referenced in #47 after SDK updates. The conflicts were pretty large so it was easier to delete and recreate my branch with these changes. Tested in my personal DevOps instance with IE, Edge, Chrome, Firefox and Safari.

joelbourbon commented:
Using moment if way too heavy to be used in Front-End.

I suggest moving to [date-fns](https://www.npmjs.com/package/date-fns) or [luxon](https://www.npmjs.com/package/luxon)

kyledodge commented:
> moment if way too heavy to be used in Front-End.
> 
> I suggest moving to [date-fns](https://www.npmjs.com/package/date-fns) or [luxon](https://www.npmjs.com/package/luxon)

I noticed just yesterday the new devops library now has a time component that does very similar functionality: https://developer.microsoft.com/en-us/azure-devops/components/time

I removed moment and changed to use the built in time ago that will also match the other devops pages

nshenoy commented:
This is now live in v1.1.2. Thank for contributing!

![Thanks](http://www.reactiongifs.com/r/thnku.gif)
